health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/utils/appConfigHelpers.js

```
@@ -142,84 +142,82 @@
-          order: field?.order,
-          readOnly: field?.readOnly || false,
-          systemDate: field?.systemDate || false,
-          pattern: field?.validations?.find((i) => i?.type === "pattern"),
-          RegexPattern: field?.validations?.find((i) => i?.type === "pattern") ? true : false,
-          MdmsDropdown: field?.schemaCode ? true : false,
-          isMdms: field?.schemaCode ? true : false,
-          isMultiSelect: field?.isMultiSelect ? true : false,
-          ...getTypeAndMetaData(field, fieldTypeMasterData),
-          ...flattenValidationsToField(field?.validations || []),
-        }));
+  return [...data1?.sort((a, b) => a.order - b.order)].map((page) => {
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)